docs/en/04_Changelogs/4.9.0.md

```diff
@@ -1,6 +1,27 @@
 # 4.9.0 (Unreleased)
 
 
-## New features
+## Features and enhancements {#features-and-enhancements}
+
+### Image lazy-loading set by default
+
+HTML `<img>` tags now have the `loading="lazy"` attribute added by default on image that are:
+- Inserted via the WYSIWYG editor
+- Added to a silverstripe template as an image object e.g. `$MyImage.SetWidth(300)`
+
+Other `<img>` tags such as those hardcoded in silverstripe templates and HTML returned in API responses remain unchanged.
+
+Individual images can have lazy-loading disabled via the following methods:
+- WYSIWYG images - Click on the image, then click the "Edit image" icon.  Modify the value in form control in the image placement form
```

Comment:
```suggestion
- WYSIWYG images - Click on the image, then click the "Edit image" icon.  Modify the lazy-loading option in the image placement form
```

Reply:
We haven't yet determined what the text for this will be, it may be 'lazy-loading', it may be 'eager-loading', it may be something else.  We can update this once the UX component has been desigend.  I'll add a point to the [UX card](https://github.com/silverstripe/silverstripe-asset-admin/issues/1214)